Cargo.toml

```diff
@@ -10,22 +10,23 @@ homepage = "https://github.com/elkowar/eww"
 
 
 [features]
-default = ["x11"]
+default = ["wayland"]
 x11 = ["gdkx11", "x11rb"]
-no-x11 = []
-
+wayland = ["gtk-layer-shell-rs"]
+no-x11-wayland = []
 
 [dependencies]
-gtk = { version = "0.9", features = [ "v3_16" ] }
-gdk = { version = "", features = ["v3_16"] }
-gio = { version = "", features = ["v2_44"] }
+gtk = { git = "https://github.com/gtk-rs/gtk-rs" }
+gdk = { git = "https://github.com/gtk-rs/gtk-rs", features = ["v3_22"] }
+gdk-sys = { git = "https://github.com/gtk-rs/gtk-rs" }
+gio = { git = "https://github.com/gtk-rs/gtk-rs" }
```

Comment:
are the git deps here really necessary still? 😢

Reply:
I think so because gtk-layer-shell-rs only version is git (it's not on crates.io)  so it is most likely based on the latest versions of gtk/gdk libs.

Reply:
If you specify the rev it may save you some pain in the future in case somethings breaks between commits.

Reply:
^ in this case, we should definitely specify the git revision explicitly!  @snakedye